SuperCollider/SuperCollider.pkg.recipe

```diff
@@ -2,94 +2,108 @@
-				<key>overwrite</key>
-				<true/>
-			</dict>
-		</dict>
-		<dict>
-			<key>Processor</key>
-			<string>PkgCreator</string>
-			<key>Arguments</key>
-			<dict>
-				<key>pkg_request</key>
-				<dict>
-					<key>pkgname</key>
-					<string>%NAME%-%version%</string>
-					<key>pkgdir</key>
-					<string>%RECIPE_CACHE_DIR%</string>
-					<key>id</key>
-					<string>%BUNDLE_ID%</string>
-					<key>version</key>
-					<string>%version%</string>
-					<key>options</key>
-					<string>purge_ds_store</string>
-					<key>chown</key>
-					<array>
-						<dict>
-							<key>path</key>
-							<string>Applications</string>
-							<key>user</key>
-							<string>root</string>
-							<key>group</key>
-							<string>admin</string>
-						</dict>
-					</array>
-				</dict>
-			</dict>
-		</dict>
-		<dict>
-			<key>Processor</key>
-			<string>PathDeleter</string>
-			<key>Comment</key>
-			<string>Clean-up working directory.</string>
-			<key>Arguments</key>
-			<dict>
-				<key>path_list</key>
-				<string>%pkgroot%</string>
-			</dict>
-		</dict>
-	</array>
+    <key>Comment</key>
+    <string>Created by James Carpenter</string>
+    <key>Description</key>
+    <string>Creates a package for SuperCollider.</string>
+    <key>Identifier</key>
+    <string>com.github.orbsmiv.pkg.SuperCollider</string>
+    <key>Input</key>
+    <dict>
+        <key>NAME</key>
+        <string>SuperCollider</string>
+        <key>BUNDLE_ID</key>
+        <string>net.sourceforge.supercollider</string>
+    </dict>
+    <key>MinimumVersion</key>
+    <string>0.6.1</string>
+    <key>ParentRecipe</key>
+    <string>com.github.orbsmiv.download.SuperCollider</string>
+    <key>Process</key>
+    <array>
+        <dict>
+            <key>Processor</key>
+            <string>PlistReader</string>
+            <key>Arguments</key>
+            <dict>
+                <key>info_path</key>
+                <string>%pathname%/SuperCollider.app/Contents/Info.plist</string>
+                <key>plist_keys</key>
+                <dict>
+                    <key>CFBundleShortVersionString</key>
+                    <string>version</string>
+                </dict>
+            </dict>
+        </dict>
+        <dict>
+            <key>Processor</key>
+            <string>PkgRootCreator</string>
+            <key>Arguments</key>
+            <dict>
+                <key>pkgroot</key>
+                <string>%RECIPE_CACHE_DIR%/pkgroot</string>
+                <key>pkgdirs</key>
+                <dict>
+                    <key>Applications</key>
+                    <string>0755</string>
+                </dict>
+            </dict>
+        </dict>
+        <dict>
+            <key>Processor</key>
+            <string>Copier</string>
+            <key>Arguments</key>
+            <dict>
+                <key>source_path</key>
+                <string>%RECIPE_CACHE_DIR%/%NAME%</string>
+                <key>destination_path</key>
+                <string>%pkgroot%/Applications</string>
```

Comment:
@macmule This bit doesn't work since it requires the previous zip/unpack bit of the download recipe. (I'm guessing that you had the previous unpack directory lurking in your cache, which is maybe why your run didn't fail.)
Suggestion below, although there's probably a neater way to do it!

```suggestion
                <key>source_path</key>
                <string>%pathname%/SuperCollider.app</string>
                <key>destination_path</key>
                <string>%pkgroot%/Applications/SuperCollider.app</string>
```